index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
+  <title>Trekitup</title>
+  <link rel="stylesheet" href="html.css">
+</head>
+<body class="grid-container">
+
+  <header>
+    <h1>Get Trekkin!</h1>
+    <div id="status-message">
+
+    </div>
+
+  </header>
+
+  <main>
+    <section class="list parallax">
+      <h1 id="load">See all trips</h1>
+      <ul id="tripList">
+
+      </ul>
+    </section>
+
+    <aside id="right-aside">
+      <section id="hide">
```

Comment:
I would use a more semantic id for this that would better indicate what data this contains, even if the ultimate goal is to hide it